ec2-spot-elastic-inference/ec2_fleet.json

```diff
@@ -6,21 +6,21 @@
   "LaunchTemplateConfigs": [
     {
       "LaunchTemplateSpecification": {
-        "LaunchTemplateName": "EI_spot",
+        "LaunchTemplateName": "EC2SpotElasticInference-LT",
         "Version": "1"
       },
       "Overrides": [
         {
           "InstanceType": "r4.xlarge",
-          "SubnetId": "%publicSubnet%"
+          "SubnetId": "subnet-0d32e94e2269a4dc4"
         },
         {
           "InstanceType": "c5.xlarge",
-          "SubnetId": "%publicSubnet%"
+          "SubnetId": "subnet-0d32e94e2269a4dc4"
```

Comment:
This is specific to your account, we should keep the placeholders. Please don't change those